r2-netty/src/main/java/com/linkedin/r2/netty/client/http2/Http2ChannelPoolFactory.java

```diff
@@ -124,7 +124,7 @@ public Http2ChannelPoolFactory(
 
     if (!StringUtils.isEmpty(_udsAddress)) {
       _bootstrap = new Bootstrap().
-          group(new EpollEventLoopGroup()).
+          group(eventLoopGroup).
           channel(EpollDomainSocketChannel.class).
```

Comment:
Same as above.

Reply:
In this case, where do we use the `EPollEventGroupLoop`? Is the change to inject that type where appropriate coming in a later PR, or being left to downstream consumers?

Reply:
It turns out that I do need to use `EpollEventLoopGroup`. Without this the Unix socket transport does not get instantiated properly. I have the container changes here -> https://linkedin.ghe.com/multiproduct/container/compare/uds-support?expand=1